mitmproxy/contentviews/image/view.py

```diff
@@ -37,19 +33,3 @@ def __call__(self, data, **metadata):
             parts = image_parser.parse_jpeg(data)
             fmt = base.format_dict(multidict.MultiDict(parts))
             return "%s image" % f, fmt
```

Comment:
Currently only JPEG, PNG, and GIF are supported and in the content-type filter. No other image types should generate this view.